### ISSUE TITLE
How to save obj?

### ISSUE THREAD

**Issue by qiuyuzhao** [philgras/neural-head-avatars#32]
from synthesize_novel_poses_and_expressions get rgba and shaded_mesh and see the image. but how to convert to obj?

rgba, shaded_mesh = synthesize_novel_poses_and_expressions(expr=expr, pose=pose, image_size=(512, 512))
verts, faces, normals, values = marching_cubes(rgba[0, :3].detach().cpu().permute(1, 2, 0).numpy())
save_obj("rgb.obj",torch.tensor(verts.copy()), torch.tensor(faces.copy()))

I convert this way, but the result is not right

**Comment by yangqing-yq** [comment]
@qiuyuzhao have you figured that out?
